devops/pollers/shared/.env.example

```
@@ -1,14 +1,19 @@
-# sqld database (self-hosted libSQL)
-# For Docker (home-poller): http://staktrakr-sqld:8080
-# For Fly.io (remote-poller via Tailscale): http://192.168.1.81:8080
-TURSO_DATABASE_URL=http://staktrakr-sqld:8080
-TURSO_AUTH_TOKEN=your-auth-token
-
-# Nightly DR sync to Turso Cloud free tier (turso-backup-sync.js)
+# Primary database — choose ONE option below.
+#
+# Option A (default): local self-hosted sqld
 SQLD_URL=http://staktrakr-sqld:8080
+SQLD_AUTH_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SQLD_AUTH_TOKEN key should go before the SQLD_URL key

(UnorderedKey)